rfs_test/TEST_accountservice_schema.py

```diff
@@ -29,272 +32,449 @@
-                        log.assertion_log('line', "Assertion Failed")  
+                        log.assertion_log('line', "Assertion Failed")
                         return assertion_status
-            
-    log.assertion_log(assertion_status, None)
-    log.assertion_log('line', "Assertion Passes") 
+
+    log.assertion_log(assertion_status, log.PASS)
+    log.assertion_log('line', "Assertion Passes")
 
     return assertion_status
-    
-## end Assertion_ACCO101
+
+# end Assertion_ACCO101
 
 ###################################################################################################
-# Name: Assertion_ACCO102(self, log) :Account Service                                            
-# Assertion text: 
-# This property shall reference the threshold for when an authorization failure is logged.  This 
-#represents a modulo function value, thus the failure shall be logged every nth occurrence where 
-#n represents the value of this property.
+# Name: Assertion_ACCO102(self, log) :Account Service
+# Assertion text:
+# This property shall reference the threshold for when an authorization failure is logged.  This
+# represents a modulo function value, thus the failure shall be logged every nth occurrence where
+# n represents the value of this property.
 ###################################################################################################
 
-'''
-Step 1: Simulate an authorization failure by providing wrong credentials.
-Step 2: Check at which attempt the authorization failure is logged.
-Step 3: Compare the value obtained from STEP 2, with the value extracted from the key "minimum" of 
-the property AuthFailureLoggingThreshold in the JSON Schema file.
-Step 4: Fail the assertion if the values at STEP 3 does not match.
-
-Concerns: What if the value AuthFailureLoggingThreshold is very large, thus requiring a considerable 
-amount of time to test the assertion. 
-'''
-## end Assertion_ACCO102
+
+def Assertion_ACCO102(self, log):
+
+    log.AssertionID = 'ACCO102'
+    assertion_status = log.PASS
+    log.assertion_log('BEGIN_ASSERTION', None)
+
+    try:
+        authorization = 'on'
+        json_payload, headers, status = self.http_GET('/redfish/v1/AccountService', rq_headers, authorization)
+        authFailureThreshold = json_payload['AuthFailureLoggingThreshold']
+        attempt = 0
+        authFailureisLogged = False; 
+
+        while(not authFailureisLogged):
+            try: 
+                authorization = 'off'
+                json_payload, headers, status = self.http_GET('/redfish/v1/AccountService', rq_headers, authorization)
+            except:
+                  print("~ \'AccountsService\' not found in the payload from GET %s" % ('/redfish/v1/AccountService'))
+
+            try: 
+                authorization = 'on'
+                json_payload, headers, status = self.http_GET('/redfish/v1/Managers/MultiBladeBMC/LogServices/Log/Entries/', rq_headers, authorization)
```

Comment:
Two things:
1) The resource being accessed will likely not be present
2) The test should be giving a bad authorization header (valid user name and a bad password)